ls1intum/Artemis
# `General`: Fix an issue with routing URLs outside of the main web page

### Checklist
#### General




### Motivation and Context


The service worker interferes with some of the server routes causing issues. An example is 
#4504 where the user cannot open the merged PDF file of all lecture's PDFs. Therefore, we need to define routes that the service worked should exclude. 

### Description

This PR defines the routes in the service worker configuration that belong to the server application.

Fixes #4504

Fixes #4506 

### Steps for Testing

Prerequisites:
1. Log in to Artemis
2. Navigate to Course Administration
3. Find a lecture with at least one PDF file unit
4. Click "Download PDF" 
5. You should be able to access the merged PDF file

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots
